docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   eventstore:
-    image: ghcr.io/eventstore/eventstore:ci
+    image: eventstore/eventstore:21.10.9-alpha-arm64v8
```

Comment:
This is meant to follow their master builds so we can test against the latest.

Also it looks like this is selecting arm64 CPU architecture? Won't this fail to run on x86_64 CPUs?

Reply:
Sorry about that, I accidentally committed that, it was for testing it locally on my macbook m1.